static/docs/use-cases/data-storage-on-external-drive.md

```diff
@@ -0,0 +1,207 @@
+# Data Storage On External Hard Drive
+
+Sometimes the data may be stored on an
+[external hard drive](https://whatis.techtarget.com/definition/external-hard-drive).
+Usually such data are huge, which means that they won't fit on our home
+directory, and even if they did, it would certainly take a long time to copy
+them back and forth from the external drive to the internal one. For example
+let's say that the size of the external drive is 16TB, while the hard drive of
+our home directory is only 320GB.
+
+In this case we would like to process the data where they are located (on the
```

Comment:
"data where they are located" -> "data where it's already located"

Reply:
👍

Reply:
Adding "already".